modules/core/src/main/java/org/mycontroller/standalone/db/SensorUtils.java

```diff
@@ -103,6 +103,7 @@ public static String getValue(SensorVariable sensorVariable) {
             case V_STATUS:
                 data = McUtils.getStatusAsString(sensorVariable.getValue());
                 break;
+            case V_BRIGHTNESS:
```

Comment:
@jkandasa I change to level because "light level" formatted as a pourcentage value. 
With level the value is formatted as double (24.0 inside of 24). Is it correct?